ExcelForce/Forms/Common/SearchSortExpressionForm.cs

```diff
@@ -44,6 +45,27 @@ public SearchSortExpressionForm(SearchSortExtractionModel model)
                 ?.RelationshipFields;
         }
 
+        public SearchSortExpressionForm(SearchSortExtractionModel model, Boolean isUpdate)
+        {
+            InitializeComponent();
+
+            searchConditionTextBox.Text = model?.SearchExpression ?? string.Empty;
```

Comment:
In future dev tasks, please move all control assignment code to separate methods.